pkg/executor/adapter.go

```diff
@@ -1678,6 +1678,10 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool, hasMoreResults bool) {
 		}
 	}
 
+	if !vardef.GlobalSlowLogRateLimiter.Allow() {
```

Comment:
Do we need to add a skip slow log-related log?

Reply:
Maybe a log the info for every 100 tries.